pkg/trace/api/api.go

```diff
@@ -444,6 +446,11 @@ func tracesFromSpans(spans []pb.Span) pb.Traces {
 	return traces
 }
 
+func getMediaType(req *http.Request) string {
```

Comment:
Since this has a chance to fail, let's return the `error` too and leave it to the caller to actually ignore it (if we want to ignore it).

Reply:
Done. Not 100% sure it adds much benefit.